packages/create-payload-app/src/main.ts

```diff
@@ -38,6 +39,8 @@ export class Main {
         '--db': String,
         '--db-accept-recommended': Boolean,
         '--db-connection-string': String,
+        '--example': String,
+        '--example-branch': String,
```

Comment:
There is already a `--template-branch`. We should consider merging these together into something simple like `--branch`.